TACC/Core-Styles
# fix: remove border from modal footer

## Overview

Remove border from Bootstrap modal footer.

## Related

- fixes "outside this pr" issue of https://github.com/TACC/tup-ui/pull/246#issuecomment-1623986198

## Changes

- **changed** `bootstrap.modal.css` to not have top border

## Testing

0. `npm start`
1. Open http://localhost:3001/components/detail/bootstrap--modal.
2. Verify modal footer does not have a border.

## UI

| before | after |
| - | - |
| <img width="1240" alt="before" src="https://github.com/TACC/Core-Styles/assets/62723358/190e4110-806a-45f0-b404-20e9d18212e1"> | <img width="1240" alt="after" src="https://github.com/TACC/Core-Styles/assets/62723358/e0d7392c-f220-485f-8f3c-dd1fe20f209b"> |
